src/main/scala/org/bitcoins/core/protocol/script/ScriptSignature.scala

```diff
@@ -190,7 +190,7 @@ object P2SHScriptSignature extends ScriptFactory[P2SHScriptSignature]  {
 
 
   def fromAsm(asm: Seq[ScriptToken]): P2SHScriptSignature = {
-    buildScript(asm, P2SHScriptSignatureImpl(_),isP2SHScriptSig(_), "Given asm tokens are not a p2sh scriptSig, got: " + asm)
+    buildScript(asm, P2SHScriptSignatureImpl(_),{ _ => true}, "Given asm tokens are not a p2sh scriptSig, got: " + asm)
```

Comment:
Need to add a comment around here to add why we are passing this trivially true function `{ _ => true }` here